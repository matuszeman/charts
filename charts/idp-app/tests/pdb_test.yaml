suite: pdb
templates:
  - pdb.yaml
values:
  - values/minimum.yaml
tests:
  - it: should match snapshot with values-ci.yaml
    values:
      - ../values-ci.yaml
    asserts:
      - matchSnapshot: {}

  - it: when multi deployment
    values:
      - values/deployment.yaml
      - values/multi.yaml
    asserts:
      - matchSnapshot: {}
      - equal:
          path: metadata.name
          value: RELEASE-NAME-one
        documentIndex: 0
      - equal:
          path: metadata.name
          value: RELEASE-NAME-two
        documentIndex: 1