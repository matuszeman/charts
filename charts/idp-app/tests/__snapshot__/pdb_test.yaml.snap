should match snapshot with values-ci.yaml:
  1: |
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:
      name: RELEASE-NAME-resource-suffix
    spec:
      maxUnavailable: 0
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME-resource-suffix
          app.kubernetes.io/name: app-name
when multi deployment:
  1: |
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:
      name: RELEASE-NAME-one
    spec:
      minAvailable: 50%
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME-one
          app.kubernetes.io/name: app-name
  2: |
    apiVersion: policy/v1
    kind: PodDisruptionBudget
    metadata:
      name: RELEASE-NAME-two
    spec:
      minAvailable: 50%
      selector:
        matchLabels:
          app.kubernetes.io/instance: RELEASE-NAME-two
          app.kubernetes.io/name: app-name
